.cirrus.yml

```diff
@@ -1,7 +1,7 @@
 task:
   name: Build and Test
   container:
-    image: rust:slim-bookworm
```

Comment:
So for this one, the build would fail on compiling mlua. Specifally the `mlua-vendored` sub crate. It fails with a file not found error. I have not been able to find yet specifically what file was missing but there is some build dep that is not there. Switching to a thicker debian image seemed to fix it.

Let me actually do some digging and find the missing dependency and add it. That's the proper way to do this

Reply:
Turns that the thing that was essential to make mlau build was `build-essential`. Who would have guessed

Updated the dockerfile and the ci